readme.md

```diff
@@ -24,6 +24,7 @@ This is list of Awesome projects using Iterative.AI tools from the Community. It
 ## 📷 🎥 Unstructured Images/video Data 
 
 ### DVC Only
+[sparrowml/mnist-pipeline](https://github.com/sparrowml/mnist-pipeline)
```

Comment:
I am using CML [a little bit too](https://github.com/sparrowml/mnist-pipeline/blob/89f04f5276289dbf46b03d6794c635d4f636b2f0/.github/workflows/pipeline.yaml#L30).

Happy to make some updates if you all have feedback on this usage!

Reply:
Oh good.  Let's move it under the correct section then!